### PR TITLE
don't enable service if it already exists

### LINKED ISSUE #17313: ubuntu upgrade re-enabled openliberty@defaultServer
**Describe the bug**

Upgaded from 21.0.0.1-1ubuntu1 to 21.0.0.5-1ubuntu1.  and `openliberty@defaultServer` was unexpectedly re-enabled.

```shell
# apt-get upgrade
…
The following packages will be upgraded:
  openliberty
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/171 MB of archives.
After this operation, 11.1 MB of additional disk space will be used.
Do you want to continue? [Y/n]
(Reading database ... 311867 files and directories currently installed.)
Preparing to unpack .../openliberty_21.0.0.5-1ubuntu1_all.deb ...
Unpacking openliberty (21.0.0.5-1ubuntu1) over (21.0.0.1-1ubuntu1) ...
Removing leftover license and run directories
Reloading systemctl services after uninstall of openliberty
Setting up openliberty (21.0.0.5-1ubuntu1) ...
Adding group - openliberty
Adding user - openliberty
Setting up the systemd openliberty service
Created symlink /etc/systemd/system/multi-user.target.wants/openliberty@defaultServer.service → /lib/systemd/system/openliberty@.service.
Creating the license directory
Creating server,log and run directory
Setting up defaultServer for user
Setting permissions to files/folders
Processing triggers for systemd (237-3ubuntu10.47) ...
Processing triggers for ureadahead (0.100.0-21) ...
root@st:/var/log/openliberty/cldr# service open
open-iscsi                 open-vm-tools              openliberty                openliberty@cldr           openliberty@defaultServer
root@st:/var/log/openliberty/cldr# service open^C
root@st:/var/log/openliberty/cldr# systemctl status openliberty@defaultServer.service
● openliberty@defaultServer.service - Open Liberty server
   Loaded: loaded (/lib/systemd/system/openliberty@.service; indirect; vendor preset: enabled)
   Active: inactive (dead)

May 26 18:46:54 st.unicode.org systemd[1]: Starting Open Liberty server...
May 26 18:46:55 st.unicode.org openliberty[1468]: Starting server defaultServer.
May 26 18:47:02 st.unicode.org openliberty[1468]: Server defaultServer started with process ID 1643.
May 26 18:47:02 st.unicode.org systemd[1]: Started Open Liberty server.
May 26 19:14:08 st.unicode.org systemd[1]: Stopping Open Liberty server...
May 26 19:14:08 st.unicode.org openliberty[11822]: Stopping server defaultServer.
May 26 19:14:10 st.unicode.org openliberty[11822]: Server defaultServer stopped.
May 26 19:14:10 st.unicode.org systemd[1]: Stopped Open Liberty server.
root@st:/var/log/openliberty/cldr# systemctl disable openliberty@defaultServer.service
Removed /etc/systemd/system/multi-user.target.wants/openliberty@defaultServer.service.
root@st:/var/log/openliberty/cldr# systemctl disable openliberty@defaultServer.service
```

**Steps to Reproduce**
- install `openliberty@21.0.0.5-1ubuntu1`
- disable the `openliberty@defaultServer` service, because we have our own named `cldr` (which automatically gets the label `openliberty@cldr.service`)
- upgrade openliberty

**Expected behavior**
- expected  openliberty@defaultServer to remain disabled, but instead it was re-enabled (and was serving pages instead of our production server)
 
**Diagnostic information:**
 - OpenLiberty Version: 21.0.0.1-1ubuntu1 to 21.0.0.5-1ubuntu1
 - Ubuntu 18.04.5 LTS amd64

**Additional context**

### PR DESCRIPTION
**Pull request by bensonlatibm**
This PR fixes release bug https://github.com/OpenLiberty/open-liberty/issues/17313

### CODE REVIEW

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#run-libby-bot

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#run-libby-bot

**Review by jjiwooLim — APPROVED** [review]
LGTM!

File: dev/build.image/packaging/debuild/openliberty/debian/postinst [hunk 1]
```diff
@@ -15,8 +15,12 @@ case "$1" in
 
         echo "Setting up the systemd openliberty service"
         if [ -x "/etc/init.d/openliberty" ] && [ -r "/lib/systemd/system/openliberty@.service" ]; then
-            systemctl daemon-reload
-            systemctl enable openliberty@defaultServer.service
+            #only enable defaultServer service if any openliberty@ services exist.
```
Comment (srl295) [inline dev/build.image/packaging/debuild/openliberty/debian/postinst:18]: Post merge, but doesn't this mean "if NO openliberty@ servies exist?" not any?

**Pull request merged**